Makefile

```diff
@@ -119,7 +121,12 @@ container: .container-$(DOTFILE_IMAGE) container-name
 	    -e 's|{ARG_OS}|$(OS)|g' \
 	    -e 's|{ARG_FROM}|$(BASEIMAGE)|g' \
 	    Dockerfile.in > .dockerfile-$(OS)_$(ARCH)
-	@docker build -t $(IMAGE):$(TAG) -f .dockerfile-$(OS)_$(ARCH) .
+	@docker build \
+	    --build-arg HTTP_PROXY=$(HTTP_PROXY) \
```

Comment:
These 2 lines are very different - on purpose?  It doesn't seem to be..

>	    --build-arg HTTP_PROXY=$(HTTP_PROXY) \

defines KEY=VAL

>	    --build-arg=$(HTTPS_PROXY) \

defines just KEY